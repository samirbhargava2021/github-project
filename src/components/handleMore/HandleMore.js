import React from "react";
import Button from '../../commonComponents/button/Button';
import { CMS } from '../../cms/CMS';

const HandleMore = ({moreItems, isNextPageExists}) => {
    return ( 
        <React.Fragment>
            {isNextPageExists && (
                <div>
                    <Button 
                        events={moreItems}
                        data={{
                            text:CMS.loadMoreRepos,
                            theme:"light"
                        }}
                        options={{
                            buttonSize:"big"
                        }}
                    />
                </div>
            )}
        </React.Fragment>
     );
}

export default HandleMore;