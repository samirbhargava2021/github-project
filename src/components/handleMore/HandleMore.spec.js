import { shallow } from "enzyme";
import HandleMore from "./HandleMore";

describe("HandleMore Component", () => {
  it("should render HandleMore Component", () => {
    const component = shallow(<HandleMore />);
    expect(component.getElements()).toMatchSnapshot();
  });

});
