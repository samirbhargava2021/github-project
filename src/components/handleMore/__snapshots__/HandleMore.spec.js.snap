// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HandleMore Component should render HandleMore Component 1`] = `
Array [
  <React.Fragment />,
]
`;
